src/Stateless/ActivateActionBehaviour.cs

```diff
@@ -8,15 +8,16 @@ public partial class StateMachine<TState, TTrigger>
         internal abstract class ActivateActionBehaviour
         {
             readonly TState _state;
-            readonly string _actionDescription;
+            readonly MethodDescription _actionDescription;
 
-            protected ActivateActionBehaviour(TState state, string actionDescription)
+            protected ActivateActionBehaviour(TState state, MethodDescription actionDescription)
             {
                 _state = state;
                 _actionDescription = Enforce.ArgumentNotNull(actionDescription, nameof(actionDescription));
             }
 
-            internal string ActionDescription { get { return _actionDescription; } }
+            internal string ActionDescription { get { return _actionDescription.Description; } }
```

Comment:
I don't ;-) .  I left it in there for backwards compatibility.  Should I remove it?